apps/src/applab/DesignToggleRow.jsx

```diff
@@ -40,6 +41,14 @@ module.exports = React.createClass({
     this.props.handleManageAssets();
   },
 
+  handleScreenChange: function (evt) {
+    this.props.onScreenChange(evt.target.value);
+  },
+
+  componentWillReceiveProps: function (newProps) {
+    this.setState({ activeScreen: newProps.initialScreen });
```

Comment:
this makes me think that activeScreen should be a prop here instead, and be state in a parent react component. since there is no parent react component this seems fine for now.